src/config/ConfigManager.cpp

```diff
@@ -151,34 +151,39 @@ void CConfigManager::setDefaultVars() {
     configValues["debug:suppress_errors"].intValue    = 0;
     configValues["debug:watchdog_timeout"].intValue   = 5;
 
-    configValues["decoration:rounding"].intValue               = 0;
-    configValues["decoration:blur:enabled"].intValue           = 1;
-    configValues["decoration:blur:size"].intValue              = 8;
-    configValues["decoration:blur:passes"].intValue            = 1;
-    configValues["decoration:blur:ignore_opacity"].intValue    = 0;
-    configValues["decoration:blur:new_optimizations"].intValue = 1;
-    configValues["decoration:blur:xray"].intValue              = 0;
-    configValues["decoration:blur:noise"].floatValue           = 0.0117;
-    configValues["decoration:blur:contrast"].floatValue        = 0.8916;
-    configValues["decoration:blur:brightness"].floatValue      = 0.8172;
-    configValues["decoration:blur:special"].intValue           = 0;
-    configValues["decoration:active_opacity"].floatValue       = 1;
-    configValues["decoration:inactive_opacity"].floatValue     = 1;
-    configValues["decoration:fullscreen_opacity"].floatValue   = 1;
-    configValues["decoration:no_blur_on_oversized"].intValue   = 0;
-    configValues["decoration:drop_shadow"].intValue            = 1;
-    configValues["decoration:shadow_range"].intValue           = 4;
-    configValues["decoration:shadow_render_power"].intValue    = 3;
-    configValues["decoration:shadow_ignore_window"].intValue   = 1;
-    configValues["decoration:shadow_offset"].vecValue          = Vector2D();
-    configValues["decoration:shadow_scale"].floatValue         = 1.f;
-    configValues["decoration:col.shadow"].intValue             = 0xee1a1a1a;
-    configValues["decoration:col.shadow_inactive"].intValue    = INT_MAX;
-    configValues["decoration:dim_inactive"].intValue           = 0;
-    configValues["decoration:dim_strength"].floatValue         = 0.5f;
-    configValues["decoration:dim_special"].floatValue          = 0.2f;
-    configValues["decoration:dim_around"].floatValue           = 0.4f;
-    configValues["decoration:screen_shader"].strValue          = STRVAL_EMPTY;
+    configValues["decoration:rounding"].intValue                = 0;
+    configValues["decoration:blur:enabled"].intValue            = 1;
+    configValues["decoration:blur:size"].intValue               = 8;
+    configValues["decoration:blur:passes"].intValue             = 1;
+    configValues["decoration:blur:ignore_opacity"].intValue     = 0;
+    configValues["decoration:blur:new_optimizations"].intValue  = 1;
+    configValues["decoration:blur:xray"].intValue               = 0;
+    configValues["decoration:blur:contrast"].floatValue         = 0.8916;
+    configValues["decoration:blur:brightness"].floatValue       = 1.0;
+    configValues["decoration:blur:boost_colors"].intValue       = 1;
+    configValues["decoration:blur:saturation_boost"].floatValue = 0.1;
```

Comment:
default should be 0. Needs more work to be default on

Reply:
Added in [932dea8](https://github.com/hyprwm/Hyprland/pull/3650/commits/932dea8d5cb783a85a84986e1ec7bcf4a8533bb6)